### PR TITLE
Update searchcommands app example

### PR DESCRIPTION
**Pull request by xuchen509**
searchcommands_app example now packages splunklib under lib folder instead of bin folder based on best practice

### CODE REVIEW

File: examples/searchcommands_app/package/lib/splunklib [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/xcheng/repo/github/splunk-sdk-python/splunklib
```
Comment (shakeelmohamed) [inline examples/searchcommands_app/package/lib/splunklib]: git ignore this file?

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (shakeelmohamed) [inline .vscode/launch.json]: git ignore this `.vscode` dir?

Comment (xuchen509) [reply .vscode/launch.json]: done

File: examples/searchcommands_app/package/lib/splunklib (hunk 1 above)
Comment (xuchen509) [reply examples/searchcommands_app/package/lib/splunklib]: done

File: tests/searchcommands/test_searchcommands_app.py [hunk 3]
```diff
@@ -415,7 +416,9 @@ def _run_command(self, name, action=None, phase=None, protocol=2):
                                 break
                             ofile.write(b)
                 with io.open(uncompressed_file, 'rb') as ifile:
-                    process = Popen(recording.get_args(command), stdin=ifile, stderr=PIPE, stdout=PIPE)
+                    env = os.environ.copy()
+                    env['PYTHONPATH'] = ":".join(sys.path)
+                    process = Popen(recording.get_args(command), stdin=ifile, stderr=PIPE, stdout=PIPE, env=env)
```
Comment (dan1) [inline tests/searchcommands/test_searchcommands_app.py]: nice :)

**Pull request merged**